packages/nextra-theme-docs/src/utils/get-git-issue-url.ts

```diff
@@ -12,16 +12,15 @@ export const getGitIssueUrl = ({
   const repo = parseGitUrl(repository)
   if (!repo) throw new Error('Invalid `docsRepositoryBase` URL!')
 
-  switch (repo.type) {
-    case 'github':
-      return `https://github.com/${repo.owner}/${
-        repo.name
-      }/issues/new?title=${encodeURIComponent(title)}&labels=${labels || ''}`
-    case 'gitlab':
-      return `https://gitlab.com/${repo.owner}/${
+  if (repo.href.includes('gitlab')) {
```

Comment:
here I would prefer check `repo.resource` because `repo.href` can be `git@github.com:IonicaBizau/not-gitlab.git`

Reply:
Yes, works for me